### ISSUE TITLE
Design & Visuals

### ISSUE THREAD

**Issue by tuna33** [tuna33/personal-site#1]
Need to come up with an idea for how the site should look, and then populate it accordingly. Place special emphasis on accessibility & responsiveness.

**Comment by tuna33** [comment]
Found a suitable initial layout. Implementing it for laptop displays first, then adjusting for different displays, and then adding accessibility features.

**Comment by tuna33** [comment]
As of #4780d49, the basic visual ideas are in place.

Since it's heavily text based at the moment, there's not much adjustment needed to account for site responsiveness. For now it'd seem like only the Projects page may need some minor adjustments on that regard.

Once the minor visual adjustments are done, accessibility should be tested. So far I'm thinking of using a screen reader and evaluating the results from that; I should also look for others' feedback for tools/methodologies/etc to account for this.
